### PR TITLE
Prevents the anomalous crystal from making fantasy into reality

### PR DESCRIPTION
**Pull request by Ergovisavi**
There was never a check to make sure holographic items weren't recreated with the refresher crystal, which delinked them from the holodeck system. Discovered this while fucking around after killing a colossus.

I almost don't want to fix this because it's fairly hilarious but on the other hand making infinite stamina e-swords and disablers is a little ehhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh

:cl:
fix: Fixed the refresher variant of the anomalous crystal making holodeck items real
/:cl:

### CODE REVIEW

**Comment by Xhuis** [comment]
_Is this just fantasy?
Caught in a landslide
No escape from reality_

**Comment by comma** [comment]
_Open your eyes,
Look up to the skies and see,
I'm just a poor boy, I need no sympathy,
Because I'm easy come, easy go,
Little high, little low,
Any way the wind blows doesn't really matter to me, to me.
_

**Comment by Anonmare** [comment]
Probably a good idea, emagging the holodeck makes those swords do actual damage

**Comment by LarissaMayrink** [comment]
_Mama, just killed a man.
Put a gun against his head.
Pulled my trigger, now he's dead.
Mama, life had just begun.
But now I've gone and thrown it all away._

**Comment by LarissaMayrink** [comment]
@Anonmare _Probably a good idea, emagging the holodeck makes those swords do actual damage_
Yes, thats totally a good idea, 6 eswords for 8tc, sounds like a great idea.

**Comment by DoctoreMad** [comment]
@LarissaMayrink Were you not illiterate, you would realize nothing of the sort was suggested.

**Comment by KingPhilipIII** [comment]
@DoctoreMad Were you not so quick to be an ass,  you would realize Larissa was simply referring to the fact you could essentially get six swords for the cost of an emag, and was being sarcastic.

**Comment by Ergovisavi** [comment]
>Bunch of thumbs down with no actual arguments as to why this change would be bad

If people would like this functionality to stick around it would be nice to see a supporting arguments for it

**Comment by kevinz000** [comment]
@DoctoreMad 's opinion is of high value @KingPhilipIII and is to be respected.

**Comment by KingPhilipIII** [comment]
@kevinz000 Apologies. It just seemed as though he was quick to be critical of someone who was simply commenting.

**Review by Cyberboss — APPROVED** [review]

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm [hunk 1]
```diff
@@ -699,7 +699,7 @@ Difficulty: Very Hard
 		for(var/i in T)
 			if(isitem(i) && !is_type_in_typecache(i, banned_items_typecache))
 				var/obj/item/W = i
-				if(!W.admin_spawned)
+				if(!W.admin_spawned && !HAS_SECONDARY_FLAG(W, HOLOGRAM))
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: please check for the ```ABSTRACT``` flag while you are at it.

**Comment by Ergovisavi** [comment]
No longer affects abstract items now as well

**Comment by LarissaMayrink** [comment]
@DoctoreMad Looks like you are the illiterate. Emagging the holodeck ALREADY makes the swords do damage, and like so if the bug doesn't gets solved you get 6eswords for 8tc, so yes, it is a perfect idea to fix this isntead of keeping it as a /feature/.

**Pull request merged**